### ISSUE TITLE
about the the general model training detail

### ISSUE THREAD

**Issue by tairen99** [clovaai/CRAFT-pytorch#148]
Hi @YoungminBaek, @ClovaAIAdmin,

Thank you for your good work.

 If it is okay, can you share your training details on your "general model"?

For your "General model", how many images were you used for the training? (all SynthText(41G), IC13, IC17)?

What kind of GPUs machines did you use for training? How many epochs and how your setup your learning rate? and how many days did you use?

Any things we need to pay attention to if we want to train it from scratch? 

Thank you in advance!
